File: source/v20140727/api/recipients/update/_update.rb
require "omise"

Omise.api_key = "skey_test_4xs8breq3htbkj03d2x"

recipient = Omise::Recipient.retrieve("recp_test_5086xmr74vxs0ajpo78")
recipient.update({
  email: "somchai@prasert.com",
  bank_account: {
    brand: "kbank",
    number: "1234567890",
    name: "SOMCHAI PRASERT"
  }
})

File: source/v20140727/api/recipients/update/_update.php
<?php

$recipient = OmiseRecipient::retrieve('recp_test_4z6p7e0m4k40txecj5o');
$recipient->update(array(
  'name' => 'Omise Tester',
  'email' => 'tester@omise.co',
  'description' => 'Another description'
));